content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,18 @@
+title: Russian's war on Ukraine
```

Comment:
Resist to take sides. It is a war between two nations, which is bad. We should not blame one or the other. The situation requires deescalation.

Reply:
We are not politicians and can't do much in regards to deescalation. IMO the situation is pretty clear on who the aggressor is in this war and talking around that is simply unjust towards the people of the invaded nation.

Reply:
I agree with your other comments to use more neutral wording.
However, we _are_ taking sides with this post, that's what it's all about, isn't it? It's clearly Russia's invasion of Ukraine, Russia's war on Ukraine, thus I don't see why to water down the heading when we mention exacly that in the text below.

Reply:
Maybe some native speaker could comment if the wording reflects this position and isn't more offensive than needed? Even in German I would neither be competent nor familiar with describing war situations and the roles of the parties involved.

Reply:
I guess calling it an "invasion" is undisputed and more specific, so I would prefer "Russia's invasion of Ukraine".

Reply:
Done